Data/ByteString/Internal.hs

```diff
@@ -648,6 +649,25 @@ concat = \bss0 -> goLen0 bss0 bss0
    concat [x] = x
  #-}
 
+-- | Repeats given ByteString n times. More efficient than default definition.
```

Comment:
```suggestion
-- | Repeats the given ByteString n times. More efficient than the default definition.
```

Which default definition? (`stimes` has one – `times` doesn't!)

Please fix the lazy version analogously.

Reply:
Have a look wether the current version is better, I just removed the sentence about efficiency.